docs/contributing.md

```diff
@@ -37,20 +37,41 @@ as these are managed by the scripts!__
 
 ## Testing your changes
 
-Obviously all changes to Catch's code should be tested. If you added new functionality, you should add tests covering and
-showcasing it. Even if you have only made changes to Catch internals (ie you implemented some performance improvements),
-you should still test your changes.
+Obviously all changes to Catch's code should be tested. If you added new
+functionality, you should add tests covering and showcasing it. Even if you have
+only made changes to Catch internals (i.e. you implemented some performance
+improvements), you should still test your changes.
 
 This means 3 things
 
-* Compiling Catch's SelfTest project -- code that does not compile is evidently incorrect. Obviously, you are not expected to
-have access to all compilers and platforms Catch supports, Catch's CI pipeline will compile your code using supported compilers
-once you open a PR.
-* Running the SelfTest binary. There should be no unexpected failures on simple run.
-* Running Catch's approval tests. Approval tests compare current output of the SelfTest binary in various configurations against
-known good output. Catch's repository provides utility scripts `approvalTests.py` to help you with this. It needs to be passed
-the SelfTest binary compiled with your changes, like so: `$ ./scripts/approvalTests.py clang-build/SelfTest`. The output should
-be fairly self-explanatory.
+* Compiling Catch's SelfTest project:
+```
+$ cd Catch2
+$ cmake -Bcmake-build-debug -H. -DCATCH_BUILD_TESTING=ON
+$ cd cmake-build-debug/
+$ make
```

Comment:
This will only work for Unixish systems and not on Windows. To support Windows, the last two steps can be changed to `cmake --build cmake-build-debug` instead.

Also we support approvals through CTest now, so the building and testing can be done like so
```
cmake -Bcmake-build-debug -H. -DCMAKE_BUILD_TYPE=Debug
cmake --build cmake-build-debug
cd cmake-build-debug
ctest -C Debug --output-on-failure
```

The advantage of this approach is that it can be used without change on both Unixish systems and Windows.

The disadvantage is that to get better control, you need to familiarize yourself with another tool (CTest), so it is probably worth it to keep a separate guide for running the approval tests manually.